.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -26,4 +26,9 @@ Tag potential reviewers from the community or maintainers who might be intereste
 
 Your PR will be reviewed more quickly if you can figure out the right person to tag with @ -->
 
-@akshayka OR @mscolnick
+@mscolnick (FE / BE / AI)
+@dmadisetti (BE: Runtime, Caching, Fileformat, AST)
+@mantz (FE: Widgets / BE: Dependency Management, LSP)
```

Comment:
Small typo in my username. As an aside, I don't find the FE/BE delineation to be particularly useful context for my focus areas (maybe it is for others). For example, if a contributor made changes to the package manager UI, I wouldn’t want them to skip tagging me, or worse assume I’m not open to reviewing it, just because I’m listed as "BE" for that feature. 

```suggestion
@manzt (Widgets, Dependency Management, LSP)
```

If the distinction feels important to keep, I’d suggest spelling out “frontend” and “backend."